tidy3d/material_library/material_library.py

```diff
@@ -273,10 +284,20 @@ class MaterialItem2D(MaterialItem):
 
 AlAs_FernOnton1971 = VariantItem(
     medium=PoleResidue(
-        eps_inf=1,
+        eps_inf=2.0791999999870825,
         poles=[
-            ((0.0 + 1j * 6674881541314847.0), (-0.0 - 1j * 2.0304989648679764e16)),
-            ((0.0 + 1j * 68198825885555.74), (-0.0 - 1j * 64788884591277.95)),
+            (
+                (-3.4784155704918245 - 1j * 6674881528384773.0),
+                (22.52074625806345 + 1j * 2.0304989609387856e16),
+            ),
+            (
+                (-69676151586035.45 - 1j * 1701334260069428.2),
+                (0.010443365309052358 + 1j * 0.018058949307349314),
+            ),
+            (
+                (-0.30731881177988135 - 1j * 68198826070986.79),
+                (0.32823363278906764 + 1j * 64788884158076.12),
+            ),
```

Comment:
This is an interesting result. In the updated version, 2 out of the 3 poles resemble the original 2 poles. I think the original result is based on the Sellmeier formula in the page https://refractiveindex.info/?shelf=main&book=AlAs&page=Fern . What improvement do you see with the new poles?